terraform/executor/Dockerfile-0.12

```diff
@@ -12,7 +12,7 @@ ARG AWSCLI_VERSION=1.18.134
 ARG HELM2_VERSION=2.16.10
 ARG HELM3_VERSION=3.3.1
 ARG TFSWITCH_VERSION=0.8.832
-ARG TF_AIVEN_VERSIONS="1.0.19 1.0.20 1.1.0 1.1.1 1.1.2 1.1.3 1.1.4"
+ARG TF_AIVEN_VERSIONS="1.0.19 1.0.20 1.1.0 1.1.1 1.1.2 1.1.3 1.1.4 1.3.5 2.0.6"
```

Comment:
latest v1 because people couldn't have dependended on the unavailable version

latest v2 for same reason, and because they need 0.13 to download via tf